exercises/binary-search/src/example/java/BinarySearch.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.List;
+
+public class BinarySearch<T extends Comparable<T>> {
+
+    public static String ARRAY_MUST_BE_SORTED = "Array should be sorted.";
```

Comment:
I think comments are just fine in the tests too - as mentioned somewhere else in this PR, we have them when JUnit rules are implemented, so that users can find relevant context more quickly. However, as is generally considered good practice, we definitely don't want comments on every line of code just for the sake of it ;)